modules/DMZ/data_kit/dataset_insight.py

```diff
@@ -30,3 +33,31 @@ def get_prediction_type(target_column):
 def get_delimiter(path):
     with open(path, 'rb') as csvfile:
         return csv.Sniffer().sniff(csvfile.read(), delimiters=';,').delimiter
+
+
+# Gets the ratio of missing values to existing values in a dataframe. Either operates on rows or columns, depending
+# on input.
+def get_missing_ratios(pandas_data, method):
+    axis_toggle = toggle.get_axis_toggle(method)
+
+    if axis_toggle != 2:
+
+        missing_data_counts = pandas_data.isnull().sum(axis_toggle)
+        ratios = []
+
+        for i in range(0, pandas_data.shape[not axis_toggle]):
+            ratio = float(missing_data_counts[i]) / pandas_data.shape[axis_toggle]
+            ratios.append(ratio)
+        return ratios
+
+    else:
```

Comment:
For future reference it would be cleaner to do,

`return pandas_data`

rather than (remove the extra line as well),

```
else:
    return pandas data
```